src/time.cpp

```diff
@@ -99,9 +99,12 @@ namespace g3 {
       ts->tv_sec  = now.tv_sec;
       ts->tv_nsec = now.tv_usec * 1000;
       return 0;
+#elif(defined(WIN32) || defined(_WIN32) || defined(__WIN32__))
+	   return timespec_get(ts, TIME_UTC);
```

Comment:
This one was annoying m. std::timespec_get(...) doesn't exist but timespec_get(...) does